test/src/tests-persistence.ts

```diff
@@ -792,7 +815,7 @@ describe('migrations and persistence-related', () => {
     it('integration test - should remove a previous MPID as a key from cookies if new user attribute added and exceeds the size of the max cookie size', done => {
         mParticle._resetForTests(MPConfig);
         mParticle.config.useCookieStorage = true;
-        mParticle.config.maxCookieSize = 650;
+        mParticle.config.maxCookieSize = 700;
```

Comment:
I had to change this because previously we were not passing in user identities to the identity calls in this test, so identities weren't being saved to cookies. Now that identities are being saved to cookies the cookies are larger and this test was breaking.  I needed to find a larger cookie size that would allow thisi test to behave as it was previously behaving.